pkgs/development/misc/resholve/resholve-utils.nix

```diff
@@ -185,6 +185,8 @@ rec {
       # retain a reference to the base
       passthru = unresholved.passthru // {
         unresholved = unresholved;
+        # fallback attr for update bot to query our src
+        originalSrc = unresholved.src;
```

Comment:
Maybe we should document this somewhere. Looks like it could be useful for a few other use cases.

Reply:
Not needed for this PR though.

Reply:
> Maybe we should document this somewhere. Looks like it could be useful for a few other use cases.

I agree, though I'm at a bit of a loss for where since `nixpkgs-update` doesn't seem to be mentioned in the nixpkgs manual or the nixos wiki. 

I feel like there could/should probably be a whole section there, or maybe a distinct guide, that covers the _broad_ topic of propagating updates to nixpkgs. At least:
- how to write and run update scripts both bespoke or via `genericUpdater`, `nix-update-script`, and any others
- code-generation best-practices for nix-ecosystem projects that need to maintain an in-tree copy of code they develop out of tree
- notes on the existence of the update bot, how to help it update your package, etc.

I have at least opened a PR over at nixpkgs-update to make it a _little_ more discoverable: ryantm/nixpkgs-update#325